app/lib/smaxtec_api.rb

```diff
@@ -4,7 +4,7 @@ class SmaxtecApi
 
   SMAXTEC_API_EMAIL = Rails.application.secrets.smaxtec_api_email
   SMAXTEC_API_PASSWORD = Rails.application.secrets.smaxtec_api_password
-  SMAXTEC_API_BASE_URL = 'https://api-staging.smaxtec.com/api/v1'
```

Comment:
Oh yes you are right, I was too confident when comitting the branch so I didn't bother to wait for the tests :blush: